DOCS/man/lua.rst

```diff
@@ -726,7 +726,9 @@ strictly part of the guaranteed API.
 
 ``utils.getcwd()``
     Returns the directory that mpv was launched from. On error, ``nil, error``
-    is returned.
+    is returned. This is equivalent to retrieving the ``working-directory``
+    property, which is preferred because it is available to clients other than
+    scripts.
```

Comment:
okay but this is not actually an argument.
If you are reading these docs you *are* in a Lua script. Whether other clients can retrieve the same stuff is irrelevant to you. You just want it from Lua.